hr_family/models/__init__.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
@GillesTephaneMeyomesse Thanks for the contribution, can you update with Small Headers or you can remove it from __init__.py files...